src/pages/Admin/Core/Templates/Nav.tsx

```diff
@@ -32,7 +32,7 @@ export default function Nav() {
         Contracts & WASM codes
       </NavLink>
       <NavLink to={routes["registrar.update-owner"]} className={styler}>
-        Update Owner
+        Update Contract Addresses
```

Comment:
No need to change this one. 
Should be the NavLink element above instead: `Contracts & WASM codes` >> `Update Contract Addresses`
```suggestion
        Update Owner
```